src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -314,7 +314,7 @@ def run(self, args, unknown_args):
         args.playbook = 'deploy_stack.yml'
         args.use_factory_auth = True
         public_vars = environment.public_vars
-        unknown_args += ('--tags=bootstrap-users',) + get_user_arg(public_vars, unknown_args, use_factory_auth=True)
+        unknown_args += ('--tags=bootstrap-users', '--skip-tags=validate_key', ) + get_user_arg(public_vars, unknown_args, use_factory_auth=True)
```

Comment:
nit
```suggestion
        unknown_args += ('--tags=bootstrap-users', '--skip-tags=validate_key') + get_user_arg(public_vars, unknown_args, use_factory_auth=True)
```

Reply:
Sure. Corrected. Previously, `unknown_args += ('--tags=bootstrap-users',)` so I followed the same thinking that it might have to end like that.